binding-hs/binding-hs.cabal

```diff
@@ -0,0 +1,51 @@
+name:                binding-hs
```

Comment:
I've never heard of hpack before. Somehow I missed it. 
Thanks for letting me know about this, saved me a lot of time now.